src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use `*` selector for reseting styles, only for `box-sizing`. Reset default styles of particular tag if needed